_build/requirements.txt

```diff
@@ -1,2 +1,16 @@
 git+https://github.com/ansible/ansible-sign
 ansible-runner>=2.3.1
+
+ansible-core
+dumb-init
+
+ncclient
+paramiko
+pyOpenSSL
+pypsrp[kerberos,credssp]
+pywinrm[kerberos,credssp]
```

Comment:
This dependency addition supports only _Kerberos_ and _CredSSP_. 
`pywinrm` is required to support _basic_, _certificate_, and _NTLM_ auth